<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirectorWindow</class>
 <widget class="QWidget" name="DirectorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>451</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="staffTab">
    <attribute name="title">
     <string>Сотрудники</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>571</width>
       <height>331</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Сотрудники</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="staffTable">
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>22</y>
       <width>209</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Нанять сотрудника</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>42</y>
       <width>49</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Позиция</string>
     </property>
    </widget>
    <widget class="QComboBox" name="positionBox">
     <property name="geometry">
      <rect>
       <x>82</x>
       <y>42</y>
       <width>147</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="salaryBox">
     <property name="geometry">
      <rect>
       <x>82</x>
       <y>141</y>
       <width>147</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1000.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>72</y>
       <width>23</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Имя</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="passportEdit">
     <property name="geometry">
      <rect>
       <x>82</x>
       <y>107</y>
       <width>147</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>141</y>
       <width>55</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Зарплата</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>107</y>
       <width>48</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Паспорт</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="nameEdit">
     <property name="geometry">
      <rect>
       <x>82</x>
       <y>72</y>
       <width>147</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="hireButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Выбререте строку сотрудника</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fireButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Уволить сотрудника</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>370</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить изменения</string>
     </property>
    </widget>
    <widget class="QPushButton" name="salaryButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Выплатить зарплату</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="bookkeepingTab">
    <attribute name="title">
     <string>Бухгалтерия</string>
    </attribute>
    <widget class="QTableView" name="bkTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>421</width>
       <height>361</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>30</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выберите дату</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>160</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выручка за день:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="reviewEdit">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>190</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="sumbitButton">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>100</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Выбрать</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>60</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="menuTab">
    <attribute name="title">
     <string>Закупка товаров</string>
    </attribute>
    <widget class="QTableView" name="menuView">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>491</width>
       <height>351</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="quantityBox">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>40</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>40</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Количество</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Товары на складе</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buyButton">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>80</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Купить</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
